pages/common/semgrep.md

```diff
@@ -0,0 +1,17 @@
+# semgrep
+
+> Perform *static source code analysis* (SAST) on your source code to detect security vulnerabilities.
```

Comment:
```suggestion
> Perform static source code analysis (SAST) to detect security vulnerabilities.
```
We don't use `you` or `your` in our pages. Also, I think we need to repeat source code here.